PROJECT-CLEARWATER-MIB

```diff
@@ -1687,6 +1690,678 @@ sproutNonRegisterAuthenticationFailures OBJECT-TYPE
                  requests over the period."
     ::= { sproutNonRegisterAuthenticationEntry 4 }
 
+sproutICSCFIncomingSipTransactionsTable OBJECT-TYPE
+    SYNTAX SEQUENCE OF SproutICSCFIncomingSipTransactionsEntry
+    MAX-ACCESS not-accessible
+    STATUS current
+    DESCRIPTION "Counts the number of attempts, successes and failures
+                 of incoming SIP transactions for the ICSCF"
+    ::= { sprout 18 }
+
+sproutICSCFIncomingSipTransactionsEntry OBJECT-TYPE
+    SYNTAX      SproutICSCFIncomingSipTransactionsEntry
+    MAX-ACCESS  not-accessible
+    STATUS      current
+    DESCRIPTION "The number of attempts, successes and failures for
+                 incoming SIP transactions for the ICSCF"
+    INDEX       { sproutICSCFIncomingSipTransactionsTimeScope,
+                  sproutICSCFIncomingSipTransactionsRequestScope }
+    ::= { sproutICSCFIncomingSipTransactionsTable 1 }
+
+SproutICSCFIncomingSipTransactionsEntry ::= SEQUENCE
+{
+  sproutICSCFIncomingSipTransactionsTimeScope      INTEGER,
+  sproutICSCFIncomingSipTransactionsRequestScope   INTEGER,
+  sproutICSCFIncomingSipTransactionsAttempts       Unsigned32,
+  sproutICSCFIncomingSipTransactionsSuccesses      Unsigned32,
+  sproutICSCFIncomingSipTransactionsFailures       Unsigned32
+}
+
+sproutICSCFIncomingSipTransactionsTimeScope OBJECT-TYPE
+    SYNTAX      INTEGER {
+                  scopePrevious5SecondPeriod(1),
+                  scopeCurrent5MinutePeriod(2),
+                  scopePrevious5MinutePeriod(3)
+                }
+    MAX-ACCESS  not-accessible
+    STATUS      current
+    DESCRIPTION "The period over which the statistics were collected."
+    ::= { sproutICSCFIncomingSipTransactionsEntry 1 }
+
+sproutICSCFIncomingSipTransactionsRequestScope OBJECT-TYPE
+    SYNTAX      INTEGER {
+                  scopeINVITE(0),

```

Comment:
Also I *think* you could define a TEXTUAL-CONVENTION (https://tools.ietf.org/html/rfc2579) to avoid duplication here:

```
RequestType ::= TEXTUAL-CONVENTION
  STATUS current
  DESCRIPTION "???"
  SYNTAX INTEGER {
                 scopeINVITE(0),
                 scopeACK(1),
                 scopeBYE(2),
                 scopeCANCEL(3),
                 scopeOPTIONS(4),
                 ...
               }


```